### ISSUE TITLE
When can we expect to see a new version published to npm?

### ISSUE THREAD

**Issue by yamov** [react-native-community/react-native-share#470]
## Ask your Question
Dear contributors, first of all thanks for your effort in maintaining this repo. My question is when can we expect to see a new version published to npm? The last release was 5 months ago and in my opinion there are some fixes worthy of a new release.

**Comment by mikehardy** [comment]
Ironically, I was thinking the same thing yesterday as I switched back to 1.1.3 from master as I'm about to release my own app and wanted a fixed version.

Which is just to say, I was using master successfully during development and at least for the "Share.open()" case with a data URL from rn-fetch-blob it was working great...

**Comment by jgcmarins** [comment]
Hello @yamov,

Thanks for asking.
We've made few changes that would need some tests: https://github.com/react-native-community/react-native-share/compare/v1.1.3...master

After @mikehardy comment:
> Which is just to say, I was using master successfully during development and at least for the "Share.open()" case with a data URL from rn-fetch-blob it was working great...

I guess we have less things to test.

**If someone is using master branch in production, let me know if it is ok for a release.**

Also there are known bugs and few opened PRs that we still need fix/merge.

**Comment by mikehardy** [comment]
@jgcmarins - there are always known bugs unfortunately, but joking aside it sounds like you have an idea in mind of what needs to get done for a 1.2.0 (my scan of the diff was that there were features but no breaking changes), maybe a milestone you could tag specific PRs and issues with? You're take on that would be really useful since you'll have a feel for everything. Then we could focus on getting a release quickly before the pressure+uncertainty build up too much

**Comment by jgcmarins** [comment]
@mikehardy, thanks for the heads up.
I totally agree, but I am little busy this week. I'll try to take a look as soon as possible.

Another issue I'd like to address is React Native version.
Maybe it should be nice to have a release before we update React Native version.

**Comment by mikehardy** [comment]
@jgcmarins sounds good - I'm not personally in a rush though I as time between releases grow I have an equally growing anxiety if I'm in any way responsible for a release :-) (nothing special for this project, that's a general thing). 

I would minimize any unnecessary change prior to 1.2.0 (assuming that's the right SemVer) so +1 waiting on react-native version, though I'll look at that separately and if it's trivial (I think it should be?) maybe it's fine to go

**Comment by MateusAndrade** [comment]
#481

**Comment by briavicenti** [comment]
Any update on this release? I'd love to pull in a released version to use the Instagram Stories share option.

**Comment by MateusAndrade** [comment]
> Any update on this release? I'd love to pull in a released version to use the Instagram Stories share option.

I was talking with @jgcmarins about this sometime ago. For now, please use the master branch and if you find any errors make sure to open a issue

**Comment by jgcmarins** [comment]
Guys, we're almost there 🔥

**Comment by karanpratapsingh** [comment]
@jgcmarins is the new update almost ready 😍 ??

**Comment by MateusAndrade** [comment]
Finally we did It!!! 🎊🎊🎊🎊

https://twitter.com/jgcmarins/status/1132688044635643904?s=19
